json/ubuntu-24.04.json

```diff
@@ -0,0 +1,8 @@
+{
+    "uuid": "ff1f3e17-84d9-4d72-962d-571c4ceb527b",
+    "reference_label": "ubuntu-24.04",
+    "name_label": "Ubuntu Noble Numbat 24.04 (preview)",
```

Comment:
The commit message does not say it is preview, 
Presume the `preview` is expected here.

Reply:
I think it`s fine. based on historical experience, when we add a template with `preview ` label, and we will not change this template anymore, 
so I wonder if we can remove the preview as well?